### PR TITLE
create layout for home screen

### PR DESCRIPTION
**Pull request by Vanh1200**
create ui home screen : layout 
https://edu-redmine.framgia.vn/issues/5856

### CODE REVIEW

**Comment by Vanh1200** [comment]
ready

File: app/src/main/res/layout/fragment_home.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                android:id="@+id/view_pager"
+                android:layout_width="match_parent"
+                android:layout_height="@dimen/dp_200" />
+
+            <include
+                layout="@layout/suggested_layout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/dp_25" />
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_home.xml:78]: remove blank line

File: app/src/main/res/layout/genres_layout.xml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+        app:layout_constraintBottom_toBottomOf="@id/image_all_music"
+        app:layout_constraintEnd_toEndOf="@id/image_all_music"
+        app:layout_constraintStart_toStartOf="@id/image_all_music"
+        app:layout_constraintTop_toTopOf="@id/image_all_music" />
+
+    <TextView
+        android:id="@+id/text_all_music"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="All Music"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/genres_layout.xml]: hardcode

**Comment by Vanh1200** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**